docs/how_to_guides/creating_and_editing_expectations/how_to_create_expectations_while_exploring_data_in_a_notebook_without_using_a_data_context.rst

```diff
@@ -3,16 +3,227 @@
+
+3. **Explore your data and add Expectations.**
+
+    Each of the methods in step 1 will produce ``my_df``, a ``PandasDataAsset``. ``PandasDataAsset`` is a subclass of ``pandas.DataFrame``, which means that you can use all of pandas' normal methods on it.
+
+    .. code-block:: python
+
+        my_df.head()
+        my_df.Sex.value_counts()
+        my_df[my_df.Sex=="male"].head()
+        # etc., etc. 
+        
+    In addition, ``my_df`` has access to a wide array of Expectations. You can see the full list :ref:`here <Glossary of Expectations>`. By convention, every Expectation method name starts with the name ``expect_...``, so you can quickly access the full list with tab-based autocomplete:
+
+    .. image:: ../../images/expectation_autocomplete.gif
+
+    |
+
+    When you invoke an Expectation, it will immediately be validated against your data. The returned object will contain the result and a list of exceptions. This instant feedback helps you zero in on exceptions very quickly, taking a lot of the guesswork out of data exploration.
+
+    .. image:: ../../images/expectation_notebook_interactive_loop.gif
+
+    |
+
+    Hint: it's common to encounter data issues where most cases match, but you can't guarantee 100% adherence. In these cases, consider using a ``mostly`` parameter. This parameter is an option for all Expectations that are applied on a row-by-row basis, and allows you to control the level of wiggle room you want built into your data validation.
+
+    .. figure:: ../../images/interactive_mostly.gif
+
+        Note how ``success`` switches from ``false`` to ``true`` once ``mostly=.99`` is added.
+
+    |
+
+4. **Review your Expectations.**
+
+    As you run Expectations in your notebook, ``my_df`` will build up a running list of Expectations. By default, Great Expectations will recognize and replace duplicate Expectations, so that only the most recent version is stored. (See :ref:`Determining duplicate results` below for details.)
+
+    You can get the config file for your Expectations by running:
+
+    .. code-block:: python
+    
+        my_df.get_expectation_suite()
+
+    which will return an :ref:`ExpectationSuite` object.
+
+    By default, ``get_expectation_suite()`` only returns Expectations with ``success=True`` on their most recent validation. You can override this behavior with:
+    
+    .. code-block:: python
+
+        my_df.get_expectation_suite(discard_failed_expectations=False)
+
+
+5. **Save your Expectation Suite.**
+
+    Expectation Suites can be serialized as JSON objects, so you can save your Expectation Suite like this:
+
+    .. code-block:: python
+    
+        import json
+
+        with open( "my_expectation_file.json", "w") as my_file:
+            my_file.write(
+                json.dumps(my_df.get_expectation_suite().to_json_dict())
```

Comment:
Do you like using `to_json_dict` here? Other options might be to use the schema directly or to `print()` the object

Reply:
hm.

* `to_json_dict` didn't give me any concerns. I like that it's self-evident what the method is doing. I agree this is a tad clunky, but that's partly deliberate (See #1544).
* The notion of using `print()` feels off to me. I think that python users think of `print` as their "write to console" method. I'm pretty sure that using it to write elsewhere will confuse people.
* I'm not sure what you mean by "using the schema directly." Can you clarify?